R/test_result.R

```diff
@@ -18,13 +17,16 @@
 #'   \item \code{num_total}: Number of tests
 #'   \item \code{errors}: Vector of errors found
 #' }
-#' @param empty_msg A character string to display as a message if the user code is NULL.
-#' @param ... ignored
-#' @param user (Optional) student code to check against the \code{results} surrounded
+#' @param empty_msg A character string to display as a message if the user code is NULL. TODO depricated.
```

Comment:
No need to give a message.  Just remove it.

Until we're on cran or > 50 stars, I'm not apologetic on removing things that exist for a short period.

Reply:
@garrettgman question about API. should `grader` deal with users who do not submit anything (i.e., `NULL` input), or should those be handled by `learnr`?